service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,140 @@
+        <actions>
+
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <econdition field-name="grandTotal" operaor="greater" from="50"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @  6.Listing all Order Item details for the order with orderId, 100102 and orderPartSeqId, 01.-->
+    <service verb="get" noun="OrderItems">
+        <out-parameters>
+            <parameter name="orderItemList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId" value="100102"/>
+                <econdition field-name="orderPartSeqId" value="01"/>
+            </entity-find>
+        </actions>
+    </service>
+
+  <!--7 Find all Order Parts with partTotal less than equals to 20.-->
+
+    <service verb="get" noun="OrderParts">
+        <out-parameters>
+            <parameter name="orderPartDetails" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartDetails">
+                <econdition field-name="orderId" operaor="less" from="20"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @ 8.Find all Order Part records assigned to the facility with ZIRET_WH value-->
+
+    <service verb="get" noun="OrderPartData">
+        <in-parameters>
+            <parameter name="facilityId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="OrderPartData" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="OrderPartData">
+            <econdition field-name="facilityId" value="ZIRET_WH"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!--  @ 9.Finding the count of order parts for the customer Party Id, “CustJqp”.-->
+
+    <service verb="get" noun="OrderPartsCount">
+        <out-parameters>
+            <parameter name="hitCount"/>
+        </out-parameters>
+        <actions>
+            <entity-find-count entity-name="mantle.order.OrderPart" count-field="hitCount" >
+                <econdition field-name="customerPartyId" value="CustJqp"/>
+            </entity-find-count>
+        </actions>
+    </service>
+
+    <!--@10.Finding all unique Order Parts with shipmentMethodEnumId value as “ShtMthGround” and facilityId as “ZIRET_WH”.-->
+
+    <service verb="get" noun="OrderPartsInfo">
+        <out-parameters>
+            <parameter name="orderPartList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartList">
+                <econdition field-name="shipmentMethodEnumId" value="ShtMthGround"/>
+                <econdition field-name="facilityId" value="ZIRET_WH"/>
+            </entity-find>
```

Comment:
Can you check if there are multiple parts with the same value for shipmentMethodEnumId and facilityId, how can we only return the unique combination only, instead of all records?

Reply:
completed